common/changes/@microsoft/tsdoc-config/feat-docxmlnodes_2022-08-03-21-34.json

```
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/tsdoc-config",
+      "comment": "Rename various properties to from HTML to XML.",
+      "type": "major"
```

```suggestion
      "type": "minor"
```

For versions `<1.0.0` a MINOR bump is considered a breaking change. (and a SemVer range such as `^0.1.3` will not match `0.2.0`)